src/Router.php

```diff
@@ -121,16 +121,8 @@ public static function is_graphql_http_request() {
 		}
 
 		// If before 'init' check $_SERVER.
-		if ( isset( $_SERVER['SERVER_NAME'] ) && isset( $_SERVER['REQUEST_URI'] ) ) {
-			$haystack = wp_unslash( $_SERVER['SERVER_NAME'] )
-				. wp_unslash( $_SERVER['REQUEST_URI'] );
-			$needle   = site_url( self::$route );
```

Comment:
This conditional checks to see if the `/graphql` route (or whatever route GraphQL is set to resolve at) is being requested so that plugin authors can do things like: 

```
if ( is_graphql_http_request() ) {
  // do something just for graphql
} else {
  // do something for all other WP requests
}
```

This PR takes away the check to see if the route is being called, so this conditional wouldn't be doing what it's intended to be doing. 

I'm open to changing this because things clearly broke with the release, but we have to make sure the function still does what it's intended to do and check that the route is indeed the graphql route.